src/styles/gallery.scss

```diff
@@ -159,4 +159,8 @@
       @apply animate-spin rounded-full h-8 w-8 border-b-2 border-whpptActive top-2/4 right-2/4;
     }
   }
+
+  &__footer {
+    @apply w-96 m-auto;
```

Comment:
Its never a good idea to hard code width to a value, This is how we get side scrolling bars